projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-v1-4-endpoint.php

```diff
@@ -88,6 +88,12 @@
 		'date_format'                             => '(string) PHP Date-compatible date format',
 		'time_format'                             => '(string) PHP Date-compatible time format',
 		'start_of_week'                           => '(int) Starting day of week (0 = Sunday, 6 = Saturday)',
+		'woocommerce_onboarding_profile'          => '(array) woocommerce_onboarding_profile',
+		'woocommerce_store_address'               => '(string) woocommerce_store_address option',
+		'woocommerce_store_address_2'             => '(string) woocommerce_store_address_2 option',
+		'woocommerce_store_city'                  => '(string) woocommerce_store_city option',
+		'woocommerce_default_country'             => '(string) woocommerce_default_country option',
+		'woocommerce_store_postcode'              => '(string) woocommerce_store_postcode option',
```

Comment:
Should we / could we update the other versions of the settings endpoint as well (v1.2, v1.3)?

Reply:
I don't have a super strong opinion. But my thoughts are just updating v1.4 is OK, because if someone needs access to these fields it encourages the use of the most recent endpoint version. Should a scenario arise where someone is locked into a lower version somehow, but also needs access to these fields, support could be added then.

Reply:
👍 Let's merge this then.